cibuildwheel/__main__.py

```diff
@@ -156,7 +159,19 @@ def main() -> None:
     test_extras = get_option_from_environment('CIBW_TEST_EXTRAS', platform=platform, default='')
     build_verbosity_str = get_option_from_environment('CIBW_BUILD_VERBOSITY', platform=platform, default='')
 
-    build_selector = BuildSelector(build_config=build_config, skip_config=skip_config)
+    setup_py = package_dir / 'setup.py'
+    setup_cfg = package_dir / 'setup.cfg'
+    pyproject_toml = package_dir / 'pyproject.toml'
+
+    if not pyproject_toml.exists() and not setup_cfg.exists() and not setup_py.exists():
```

Comment:
Not wrong, but the previous

```
if not any((package_dir / name).exists()
           for name in ["setup.py", "setup.cfg", "pyproject.toml"]):
```
was arguably a tiny bit more complex, but also less code duplication. (I have honestly no clue who wrote it.)

Reply:
Could even be:


```python
if not any(p.exists() for p in {setup_py, setup_cfg, pyproject_toml}):
```

Don't have a strong preference, really, it's mostly different than what was there before because I had replaced it then rewrote it.

Reply:
Then you still have the repetition, though, of `package_dir /` and `setup.py`/`setup_py`.

> it's mostly different than what was there before because I had replaced it then rewrote it.

But I guessed so (and couldn't see a convincing reason for the change), so that's why I wanted to point it out.

Reply:
I went pack to something closer to what we had before.